compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -655,6 +655,12 @@ error.invalid.namespace.declaration=\
 error.cannot.update.xml.sequence=\
   cannot update an xml sequence
 
+error.operation.does.not.support.xml.navigation.access=\
+  invalid operation: type ''{0}'' does not support xml navigation access
+
+error.invalid.function.invocation.type=\
+  type ''{0}'' cannot be used to invoke function
```

Comment:
Shall we specifically say that `xml langlib functions does not support union types as their arguments`